package.json

```diff
@@ -68,6 +68,7 @@
     "@semantic-release/git": "9.0.0",
     "@types/jest": "26.0.13",
     "babel-eslint": "10.0.3",
+    "conventional-changelog-conventionalcommits": "4.5.0",
```

Comment:
I thought the only way to define a breaking change was this one:
https://github.com/semantic-release/semantic-release#commit-message-format

with a description of the breaking change in the commit message body.

Are we sure, we want to enable breaking changes to be committed without explanation (ie using `!`)?

Reply:
I think you are right on the fact that a BREAKING CHANGE should come with a description, making this contribution un-helpful. 

Let's update our commit on the `beta` branch to add a BREAKING CHANGE section in the body and re-check that the pre-release happen this time.